node/src/chain_spec.rs

```diff
@@ -166,8 +166,10 @@ pub fn amplitude_config() -> AmplitudeChainSpec {
 				// initial collators.
 				invulnerables.clone(),
 				signatories.clone(),
+				vec![sudo_account.clone()],
```

Comment:
We probably want to consider adding more authorized oracles already in the genesis config. I'll create a separate ticket for generating them but it would need to be added here.

Reply:
Nevermind, we don't care as genesis is kind of useless